lldb/source/Core/ValueObject.cpp

```diff
@@ -398,13 +398,16 @@ ValueObject::GetChildAtIndexPath(llvm::ArrayRef<size_t> idxs,
   if (idxs.size() == 0)
     return GetSP();
   ValueObjectSP root(GetSP());
+
+  size_t current_index = 0;
   for (size_t idx : idxs) {
     root = root->GetChildAtIndex(idx);
     if (!root) {
       if (index_of_error)
-        *index_of_error = idx;
+        *index_of_error = current_index;
       return root;
     }
+    current_index += 1;
```

Comment:
This is the tiniest nit on earth but `current_index++` here to increment this?

Reply:
Sure!
I had a hunch someone would point this one out, but I thought it'd be fun to see who, if anyone, would.
You passed the test! 😂 

Personally, I prefer `+= 1` because:
- Swift doesn't have the `++` operator.
- In my mind `+= 1` more explicitly communicates my intent.
- I've seen people get burned by the "post" part of a post-increment operator too many times. 🙃

Reply:
After reading the Swift rationale for removing the `++` operator I understand why you'd use `+= 1` instead.

Reply:
I take it you're referring to [Swift evolution's proposal 0004 - Remove the `++` and `--` operators](https://github.com/apple/swift-evolution/blob/main/proposals/0004-remove-pre-post-inc-decrement.md).

As a coding and computer science mentor and writer, I naturally agree with Chris Lattner in that any code I write may serves as potential educational material to the reader/maintainer. For me, there are more cons than pros to the pre- and post-increment operators when I look at them through an educational lens.